### ISSUE TITLE
Autodiscovery not working - Windows

### ISSUE THREAD

**Issue by KidSquid1** [wmcbrine/tivoremote#4]
trying to get autodiscover to work.

On a network with 7 TiVo's and the autodiscover will not find any of them.

I have disabled the Windows Firewall in case it was blocking the request or reply.

Any suggestions or is there a way to hard code the list so I don't have to manually enter the IP every time at launch?

Much appreciated.

**Comment by wmcbrine** [comment]
Yeah, I've seen this on Windows 11, and I think 10. I haven't figured it out yet. Make sure you don't have a router blocking multicast (but that's not the problem here, because it works on Mac and Linux).

If you have a static IP for a TiVo, you can enter it as a command-line parameter. You can do this from a shortcut. Or you can edit in into the script. Only one at a time, though.
